.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+
+jobs:
+  ansible-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+      - name: Install ansible-lint
+        run: pip install ansible-lint
+      - name: Run tests
+        run: |
+          ansible-lint -r rules/ playbooks/*
+          ansible-lint -r rules/ roles/*
+
+  vagrant:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: vagrant
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
```

Comment:
I think I tried to get Ruby 2.0 going as well but that's pretty much impossible nowadays in CI.

Reply:
We've been running on 2.5 before (that's what Bionic has by default) and I honestly don't care about 2.0 in CI.
If it ever breaks, it's probably easier to update the nodes we still run on 2.0 from EL7 to something more modern (via SCL or whatever).

Reply:
I think in CentOS CI we can request EL8 nodes instead. https://github.com/theforeman/foreman-infra/blob/612b8ec679506a4c3a2f3025e9655bf322945383/puppet/modules/jenkins_job_builder/files/centos.org/ansible/provision.yml#L6-L10 currently doesn't specify anything but https://github.com/redhat-openstack/ansible-role-ci-centos#role-variables suggests we can set release to 8. That would be a start.